apps/api-gateway/src/auth/auth.service.ts

```diff
@@ -0,0 +1,124 @@
+import { ForbiddenException, Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { JwtService } from '@nestjs/jwt';
+import { JwtPayload } from 'jsonwebtoken';
+import { firstValueFrom } from 'rxjs';
+import { User } from '../user/entities/user.entity';
+import { UserService } from '../user/user.service';
+import { Payload } from './types/jwt.type';
+import { GithubUser } from './types/user.type';
+
+type LoginReturn = Partial<GithubUser> & { access_token?: string; refresh_token?: string } & {
+  isExist: boolean;
+};
```

Comment:
불필요하게 intersection 타입이 사용된 것 같습니다~ 아래와 같은 느낌은 어떨까요?
```suggestion
type LoginReturn = Partial<GithubUser> & {
  isExist: boolean;
  access_token?: string; 
  refresh_token?: string
}
```

Reply:
[9f15204](https://github.com/GrebbMe/grabbme-be/pull/53/commits/9f15204951c16a6285acd07e4ca1780b9c77f8b4)
말씀하신 구조가 더 깔끔하네요!
급하게 하느라 참 깔끔하게 할 생각을 못헀습니다...
위 커밋에 반영하였습니다!